gymnasium_robotics/envs/maze/ant_maze.py

```diff
@@ -271,21 +272,27 @@ def reset(self, *, seed: Optional[int] = None, **kwargs):
 
         obs, info = self.ant_env.reset(seed=seed)
         obs_dict = self._get_obs(obs)
+        info["success"] = bool(
+            np.linalg.norm(obs_dict["achieved_goal"] - self.goal) <= 0.45
+        )
 
         return obs_dict, info
 
     def step(self, action):
         ant_obs, _, _, _, info = self.ant_env.step(action)
         obs = self._get_obs(ant_obs)
 
+        info["success"] = bool(np.linalg.norm(obs["achieved_goal"] - self.goal) <= 0.45)
```

Comment:
`info` should be after `truncated`

Reply:
I think the order doesn't matter. Correct me if I'm missing anything @Kallinteris-Andreas

Reply:
It does not affect the behavior, but makes it more readable

Reply:
Sure. @alexdavey if you can update it as per @Kallinteris-Andreas request that'll be great :)